src/LibraryViewExtensionWebView2/LibraryViewController.cs

```diff
@@ -171,6 +171,7 @@ private string ReplaceUrlWithBase64Image(string html, string minifiedURL, bool m
         {
             var ext = string.Empty;
             var magicstringprod = "__webpack_require__.p+";
+            var minfiedURLHtmlReplacement = minifiedURL;
```

Comment:
spelling - `minified`

Reply:
Fixed in the next commit: https://github.com/DynamoDS/Dynamo/pull/13417/commits/0ee6320b601fc8bf1976063b2db22dcf4855f6b2